css/footer.css

```
@@ -34,21 +34,59 @@ footer {
   }
   .footer_container {
     width: 100%;
-    padding-left: 20px;
-    padding-right: 20px;
+    padding-left: 24px;
+    padding-right: 24px;
+  }
+}
+
+@media (max-width: 768px) {
+  .banner_container {
+    margin-top: 0;
+  }
+  .footer_container {
+    padding-left: 104px;
+    padding-right: 104px;
   }
 }
 
 @media (max-width: 640px) {
   .footer_container {
-    flex-direction: column;
-    align-items: center;
+    /* flex-direction: column;
+    align-items: center; */
+    flex-wrap: wrap;
+    height: 140px;
     gap: 20px;
+    padding-left: 24px;
+    padding-right: 24px;
+  }
+
+  .footer_text {
+    order: 1;
+    width: 40%;
+    gap: 22px;
+  }
+
+  .footer_sns {
+    order: 2;
+    width: 40%;
+    justify-content: flex-end;
+  }
+
+  .footer_copy {
+    order: 3;
   }
 }
 
-@media (max-width: 320px) {
+@media (max-width: 375px) {
   footer {
-    width: 320px;
+    width: 375px;
```

💊 제안
반응형 작업을 할때 필요한 경우가 아니라면 반응형 단위를 쓰는 것이 자연스럽게 움직이는 디자인을 구현하고 코드를 줄이는데 좋습니다!
지금같이 경우 mediaQuery의 조건문을 통해 최대 너비를 375로 제한하고 있으니 아래처럼 작성하시면 더 좋을 것 같아요.
```suggestion
    width: 100%;
```